src/components/products/bubblegum/index.js

```diff
@@ -87,6 +87,10 @@ export const bubblegum = {
               title: 'How to Create a 1,000,000 NFT Collection on Solana',
               href: 'guides/javascript/how-to-create-1000000-nfts-on-solana',
             },
+            {
+              title: 'How to Interact with cNFTs on Other SVMs',
+              href: 'guides/javascript/how-to-interact-with-cnfts-on-other-svms',
```

Comment:
both of these links do not seem to work and result in something like `bubblegum/guides/javascript/guides/javascript/how-to-create-1000000-nfts-on-solana`
So the link probably should just be `guides/javascript/how-to-interact-with-cnfts-on-other-svms)`

Reply:
fixed